src/RouterSelector.js

```diff
@@ -35,7 +36,16 @@ let SignupRouter = React.lazy(() =>
  * @returns {Promise<boolean>}
  */
 const handleLinks = async () => {
-  const { params } = Linking
+  const isWeb = Platform.OS === 'web'
+  let params
+
+  if (isWeb) {
```

Comment:
we dont need a special case for web, we need to implement same interface for linking.web.js and linking.native.js